porcupine/plugins/filetypes.py

```diff
@@ -264,13 +279,13 @@ def setup() -> None:
         values=sorted(filetypes.keys(), key=str.casefold),
     )
     set_filedialog_kwargs()
-
+    global filetypes_var
+    filetypes_var = tkinter.StringVar()
     for name in sorted(filetypes.keys(), key=str.casefold):
         escaped_name = name.replace("/", "//")  # doesn't work in all corner cases
```

Comment:
We no longer need `escaped_name`, you can just use `name`. Previously, a filetype named `"Foo/bar file"` would result in `add_filetype_command("Filetypes/Foo//bar file", ...)`, and it would eventually call `.add_command(label="Foo/bar file", ...)`. But if a filetype now contains a slash, we simply add a menu item whose label contains a slash.